firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestore.java

```diff
@@ -333,37 +333,36 @@ public WriteBatch batch() {
     return batch.commit();
   }
 
-  Task<Void> shutdownInternal() {
+  Task<Void> terminateInternal() {
     // The client must be initialized to ensure that all subsequent API usage throws an exception.
     this.ensureClientConfigured();
-    return client.shutdown();
+    return client.terminate();
   }
 
   /**
-   * Shuts down this {@code FirebaseFirestore} instance.
+   * Terminates this {@code FirebaseFirestore} instance.
    *
-   * <p>After shutdown only the {@link #clearPersistence()} method may be used. Any other method
+   * <p>After terminate only the {@link #clearPersistence()} method may be used. Any other method
```

Comment:
I think this should be "termination" or maybe "calling {@code terminate()}".  I think I prefer the latter to be more explicit.

Reply:
Done.